src/js/providers/video-actions-mixin.js

```diff
@@ -21,20 +22,24 @@ const VideoActionsMixin = {
         if (!width || !height || !this.video.videoWidth || !this.video.videoHeight) {
             return false;
         }
+        const _videotag = this.video;
+        const styles = {
+            objectFit: '',
+            width: '',
+            height: '',
+        };
         if (stretching === 'uniform') {
             // snap video to edges when the difference in aspect ratio is less than 9%
             var playerAspectRatio = width / height;
-            var videoAspectRatio = this.video.videoWidth / this.video.videoHeight;
-            var objectFit = null;
+            var videoAspectRatio = _videotag.videoWidth / _videotag.videoHeight;
```

Comment:
Let's use `const` and `let` where applicable

Reply:
Changed to let in next commit

Reply:
```js
function _styleValue(property, value) {
    if (value === '' || value === undefined || value === null) {
        return '';
    }
```